.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "modules/taskcluster-terraform"]
 	path = modules/taskcluster-terraform
-	url = https://github.com/taskcluster/taskcluster-terraform
+	url = https://github.com/taskcluster/taskcluster
```

Comment:
With this change, the terraform will be at `modules/taskcluster-terraform/infrastructure/terraform` instead of just `modules/taskcluster-terraform`.  That's not a problem, but it will require some changes to the terraform in this repository to point to it.  We could also change the `path =` here to make a more logical path -- maybe `monorepo/infrastructure/terraform`?

I don't mind having the whole monorepo as a submodule (disk space is cheap!) and I don't know of any good alternatives.  Maybe @imbstack has ideas?

Reply:
Let's just have the whole monorepo as the submodule. Just buy some @seagate stock before landing this!

Reply:
Oh, I literally meant that this was good as is! Land it!

Reply:
I'm curious what the concerns are with using the whole monorepo as a submodule?